src/System.Runtime/tests/System/UriBuilderTests.cs

```diff
@@ -0,0 +1,384 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Tests
+{
+    public static class UriBuilderTests
+    {
+        [Fact]
+        public static void Ctor_Empty()
+        {
+            var uriBuilder = new UriBuilder();
+            VerifyUriBuilder(uriBuilder, "http", "", "", "localhost", -1, "/", "", "");
```

Comment:
Nit: Consider using named arguments so that it's clear which string maps to what, e.g.
```C#
VerifyUriBuilder(uriBuilder, 
    scheme: "http", host: "localhost", path: "/", query: "", fragment: "",
    userName: "", password: "",
    port: -1);
```